lib/remove_unwanted_table.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveUnwantedTable < Scraped::Response::Decorator
```

Comment:
Yep, good catch. It turns out we now longer need this now we're auto-detecting tables, since we now just ignore tables that we don't handle, so I've removed it altogether.